src/domain/cookieConsent/CookieConsent.tsx

```diff
@@ -0,0 +1,127 @@
+    () => ({
+      siteName: appName,
+      onAllConsentsGiven: () => {
+        setShowCookieConsentModal(false);
+      },
+      currentLanguage: language as string as ContentSource['currentLanguage'],
+      requiredCookies: {
+        groups: [
+          {
+            commonGroup: 'essential',
+            cookies: [
+              {
+                id: 'wordpress',
+                name: 'wordpress_*, wp-settings-*',
+                hostName: 'api.hel.fi',
+                description: t('common:consent.cookies.wordpress'),
+                expiration: t('common:consent.expiration.session'),
+              },
+              {
+                id: 'linkedevents',
+                name: 'linkedevents-api-prod-csrftoken',
+                hostName: 'api.hel.fi',
+                description: t('common:consent.cookies.linkedevents'),
+                expiration: t('common:consent.expiration.year'),
+              },
+              {
+                id: 'matomo',
+                name: '_pk*',
+                hostName: 'digia.fi',
+                description: t('common:consent.cookies.matomo'),
+                expiration: t('common:consent.expiration.days', { days: 393 }),
+              },
+              {
+                id: 'i18next',
+                name: 'i18next',
+                hostName: 'api.hel.fi',
+                description: t('common:consent.cookies.i18next'),
+                expiration: t('common:consent.expiration.session'),
+              },
+            ],
+          },
+        ],
+      },
+      optionalCookies: {
+        groups: [
+          {
+            title: t('common:consent.groups.servicemap.title'),
+            text: t('common:consent.groups.servicemap.text'),
+            expandAriaLabel: t(
+              'common:consent.groups.servicemap.expandAriaLabel'
+            ),
+            checkboxAriaDescription: t(
+              'common:consent.groups.servicemap.checkboxAriaDescription'
+            ),
+            cookies: [
+              {
+                id: 'servicemap_analytics',
+                name: '_pk.fi*',
+                hostName: 'palvelukartta.hel.fi',
+                description: t('common:consent.cookies.servicemap_analytics'),
+                expiration: t('common:consent.expiration.days', { days: 393 }),
+              },
+              {
+                id: 'servicemap_session',
+                name: 'ADRUM',
+                hostName: 'palvelukartta.hel.fi',
+                description: t('common:consent.cookies.servicemap_session'),
+                expiration: t('common:consent.expiration.session'),
+              },
+              {
+                id: 'servicemap_ga',
+                name: '_ga*',
+                hostName: 'palvelukartta.hel.fi',
+                description: t('common:consent.cookies.servicemap_ga'),
+                expiration: t('common:consent.expiration.session'),
+              },
+            ],
+          },
+        ],
+      },
```

Comment:
Only thing I found related to Palvelukartta, is a 1 link that we create, but which is also handled as an external link that opens the target in a new tab. I don't think we need to have an optional cookie option for that, do we? Because we are already indicating that it's opening in a different site. If it is clear enough, that I don't know. Anyway, we don't even have feature that would be triggered from the selection. Should the link be hidden if the cookie consent is not accepted?

Reply:
The places menu is actually using the Palvelukartta through the Unified-Search, but it's still not using it's cookies.

Reply:
yes, fixed